core/declarations.gms

```diff
@@ -48,6 +48,7 @@ f_lab(tall,all_regi,all_POPscen)                     "labour data for all possib
 pm_lab(tall,all_regi)                                "data for labour [bn people]"
 pm_esCapCost(tall,all_regi,all_teEs)                 "Capital energy cost per unit of consumption for end-use capital (energy service layer)"
 pm_cesdata_sigma(ttot,all_in)                        "elasticities of substitution"
+*' If elasticities of substitution (sigma) is below 1, the smaller it is the less the substitution replacement effect between different CES nodes. The products become more and more complementary in the production, meaning that the more one product is used, the more the other one is demanded as well. If sigma is larger than 1, the more one product is used, the less the others are used, i.e. the products are substitutes
```

Comment:
this comment should go next to an equation or somewhere else and not be in declarations.gms

Reply:
I see, is there a reason we cannot put goxygen in declarations.gms? it might be helpful to further explain the declared variables.. I can also include it in the definition..